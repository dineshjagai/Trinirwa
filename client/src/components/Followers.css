.containers {
    margin: auto;
    width: 98%;
    display: flex;
    flex-direction: row;
    flex-wrap: wrap-reverse;
    justify-content: space-evenly;
}

.box-container {
    background:white;
    border-radius: 8px;
    box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
    height: max-content;
    overflow: hidden;
}
.box {
    height: 500px;
    overflow: auto;
}

#delete:hover {background-color: indianred;}

.title {
    margin: auto;
    text-align: center;
    font-weight: bold;
}
