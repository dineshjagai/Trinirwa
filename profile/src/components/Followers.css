.containers {
    display: flex;
    flex-wrap: wrap;
    flex-direction:vertical;
    justify-content: flex-start;
    background: white;
    border-radius: 15px;
    width:25%;
    box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);

}