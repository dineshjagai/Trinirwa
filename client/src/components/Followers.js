import React from 'react';
import './Followers.css';
// import CircularProgress from '@material-ui/core/CircularProgress';
import Follower from './Follower';

export default function Followers({ input }) {
  const items = input.map((follower) => <Follower info={follower} />);
  return (
    <div className="box">
      <div className="buttonFollowers">
        <button className="btn" id="delete" style={{ color: '#00695c', border: 'none', backgroundColor: 'inherit' }} type="button"> All followers</button>
      </div>
      <div className="title">Followers</div>
      <br />
      <div className="containers">
        {items}
      </div>
    </div>
  );
}
