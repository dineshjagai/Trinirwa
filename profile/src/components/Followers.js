import React,{ useState }from 'react';
import './Followers.css';
import Follower from './Follower.js';

export default function Followers(props) {
    const[followers, setList] = useState(props.followers);
    const items = followers.map((follower)=>
        <Follower info={follower}/>
    );
    console.log(items);
    return(
        <div className="containers">
            {items}
        </div>
    );
}