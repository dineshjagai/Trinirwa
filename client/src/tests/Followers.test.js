import React from 'react';
import renderer from 'react-test-renderer';
import Followers from '../components/Followers';

test('Test Followers snapshot', () => {
  const component = renderer.create(<Followers />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
