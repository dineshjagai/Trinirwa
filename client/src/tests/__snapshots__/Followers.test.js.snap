// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Followers snapshot 1`] = `
<div
  className="box-container"
>
  <div
    className="buttonFollowers"
  >
    <div>
      <button
        className="btn"
        id="delete"
        onClick={[Function]}
        style={
          Object {
            "backgroundColor": "inherit",
            "border": "none",
            "color": "#00695c",
          }
        }
        type="button"
      >
        All followers
      </button>
    </div>
  </div>
  <div
    className="title"
  >
    Followers
  </div>
  <br />
  <div
    className="box"
  >
    <div
      className="containers"
    />
  </div>
</div>
`;
